drivers/SmartThings/samsung-audio/src/disco.lua

```diff
@@ -120,20 +126,24 @@ function Disco.find(deviceid, callback)
       local headers = process_response(val)
       local ip, port = headers["location"]:match(
                              "http://([^,/]+):([^/]+)") -- TODO : We need to check the xml filename for samsung audio device for ex: http://192.168.0.1:59666/rootDesc.xml
-      local id = headers["usn"]
-
+      -- local id = headers["usn"]
```

Comment:
If this is no longer needed, we should delete instead of comment it out.

Reply:
Done